app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,7 @@
+class StaticPagesController < ApplicationController
+  def home
```

Comment:
Method rỗng thì em viết trên một dòng thôi 
```suggestion
  def home; end
```